v1/definition.yaml

```diff
@@ -150,35 +100,7 @@ paths:
                 content-type: '{{extract.headers.content-type}}'
                 vary: '{{extract.headers.vary}}'
               body: '{{extract.body}}'
-        - emit_change_event:
```

Comment:
The `response_cache_control` here will still be pretty long. We need to either return some kind of purging or make much smaller cache max-age

Reply:
In [the CP config patch](https://gerrit.wikimedia.org/r/534122) we keep the purging side of things, so this will not be an issue.

Reply:
No. If you don't emit the resource-change for this route, no purge will be emitted.

Reply:
Indeed, I'll add a purge event to the CP config, that ought to fix it.

Reply:
@Pchelolo updated the CP config patch to reflect what we discussed here